source/_usecases/format-curreny-as-per-the-locale.md

```diff
@@ -0,0 +1,32 @@
+---
+extends: _layouts.usecase
+date: 2018-03-04
+link: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString
+category: number
+---
+
+
+```javascript
+function formatCurrency(value, locale, currency) {
+  return value.toLocaleString(locale, { style: "currency", currency: currency });
+}
+// locale can be obtained using "navigator.language"
+
+const amount = 1999.90;
+
+console.log("Netherlands: ", formatCurrency(amount, "nl-NL", "EUR"));
+console.log("German: ", formatCurrency(amount, "de-DE", "EUR"));
```

Comment:
Please only keep **one** of these examples in order to keep use case concise
Feel free to choose whichever one you prefer

Reply:
Have removed all except one.